src/app/rooms/[roomId]/page.module.scss

```
@@ -311,43 +310,41 @@
           width: 8px;
         }
 
         &::-webkit-scrollbar-track {
-          background: rgba(f.color(orange-600), 0.05);
+          background: rgba(f.color(gray-600), 0.05);
           border-radius: 999px;
         }
 
         &::-webkit-scrollbar-thumb {
-          background: rgba(f.color(orange-600), 0.3);
+          background: rgba(f.color(gray-600), 0.3);
```

Stylelint (17.2.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)